### DISCUSSION TITLE
Add support for Canon .clut 3D LUT format.

### DISCUSSION THREAD

**Discussion by TylerPruitt** [videovillage/community#22] [Ideas]
Attached is example 3D LUT file.  I have added a .txt extension to the file for upload to GitHub.  The correct extension is .clut

[unity test.clut.txt](https://github.com/videovillage/Lattice-Issues/files/3114469/unity.test.clut.txt)

**Comment by eweidt** [comment]
This would be a great feature, as the only way I know to make these is with Canon's windows-only conversion tool.
